examples/deadletterqueue/processMessagesInDLQ.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  delay,
+  OnMessage,
+  OnError,
+  MessagingError,
+  ServiceBusMessage,
+  ReceiveMode,
+  Namespace,
+  QueueClient
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const queuePath = process.env.QUEUE_NAME || "";
+const deadLetterQueuePath = queuePath + "/$DeadLetterQueue";
+const receiveClientTimeout: number = 10000;
+console.log("str: ", str);
+console.log("path: ", queuePath);
+
+let ns: Namespace;
+
+/*
+  This sample demonstrates how messages from DLQ can be retrieved, inspected and reprocessed if
+  required, by sending to the same or a different queue.
+
+  Run movingMessagesToDLQ sample before this to setup, populate the DLQ.
+*/
+async function main(): Promise<void> {
+  try {
+    ns = Namespace.createFromConnectionString(str);
+    // Process messages from the Dead Letter Queue, by invoking .deadletter() on the brokered message
+    await processMessageQueue(deadLetterMessageProcessor, deadLetterQueuePath);
+  } catch (err) {
+    console.log(">>>>> Error occurred in running sampple: ", err);
+  } finally {
+    console.log("\n>>>> Calling close....");
+    ns.close();
+  }
+}
+
+async function processMessageQueue(messageProcessor: Function, path: string): Promise<void> {
```

Comment:
Let's inline this method into the caller.

Reply:
Updated